### PR TITLE
[14.0] 53993 interests on questions

### PR DESCRIPTION
**Pull request by marielejeune**
refs #53993
@qgroulard review requested

### CODE REVIEW

File: mozaik_all/__manifest__.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
         "mozaik_event_partner_firstname",
         "mozaik_event_publish_date",
         "mozaik_event_thesaurus",
+        "mozaik_event_thesaurus_question",
```
Comment (qgroulard) [inline mozaik_all/__manifest__.py]: mozaik_event_question_thesaurus

File: mozaik_event_thesaurus_question/models/event_event.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                        "You cannot change the type of an event when there are attendees."
+                    )
+                )
+        return super(EventEvent, self).write(vals)
+
+    @api.depends("event_type_id")
+    def _compute_question_ids(self):
+        super()._compute_question_ids()
+        for event in self:
+            command = [(5, 0)]
```
Comment (qgroulard) [inline mozaik_event_thesaurus_question/models/event_event.py]: This is not compatible with `mozaik_event_tickbox_question`.

File: mozaik_all/__manifest__.py (hunk 1 above)
Comment (marielejeune) [reply mozaik_all/__manifest__.py]: Done

File: mozaik_event_thesaurus_question/models/event_event.py (hunk 2 above)
Comment (marielejeune) [reply mozaik_event_thesaurus_question/models/event_event.py]: Field is_mandatory is now explicitely copied.

**Comment by marielejeune** [comment]
@RobinetDenisAcsone review requested

File: mozaik_event_thesaurus_question/models/event_event.py (hunk 2 above)
Comment (qgroulard) [reply mozaik_event_thesaurus_question/models/event_event.py]: You can't do this without inheriting `mozaik_event_tickbox_question`.
But I'm not sure it's the way to go... This is a bit complex and need more thinking.

**Pull request merged**